src/components/detail/Detail.js

```diff
@@ -81,14 +81,16 @@ class Detail extends React.Component {
                 errorMessage={props.errorMessage}
               />}
             </div>
+            {props.isForgotPasswordSuccessful && <div className="alert alert-success" role="alert">{strings.RESET_PASSWORD_FOR_THIS_USER_SUCCESS}</div>}
             {props.isUpdated && state.isSaved && <div className="alert alert-success" role="alert">{strings.UPDATE_SUCCESS}</div>}
             {props.isError && <div className="alert alert-danger" role="alert">{props.errorMessage}</div>}
-            {props.showDeprecationStatus && props.isDeprecated && <div className="alert alert-danger" role="alert">{strings.formatString(strings.DEPRECATED_SUCCESS, { item: <strong>{props.title}</strong> })}</div>}
-            {props.showDeprecationStatus && !props.isDeprecated && <div className="alert alert-info" role="alert">{strings.formatString(strings.UNDEPRECATED_SUCCESS, { item: <strong>{props.title}</strong> })}</div>}
+            {props.showDeprecationStatus && props.isDeprecated && <div className="alert alert-success" role="alert">{strings.formatString(strings.DEPRECATED_SUCCESS, { item: <strong>{props.title}</strong> })}</div>}
```

Comment:
Line 87 and 88 are also duplicates
Change the message according to the props and nog the whole div